polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -197,6 +216,10 @@ $disabled-fade: 0.6;
   &:not(:first-child) {
     .ItemInnerWrapper {
       border-top: var(--p-border-width-1) solid var(--p-color-bg-app);
+
+      #{$se23} & {
+        border-top: 0;
```

Comment:
I remember having the border top added a 1px gap between nav items in a way that didn't cause a flicker of cursor changes when moving down the nav items. I think what we might want here is 

```suggestion
        border-top: var(--p-border-width-1) solid transparent;
```
Or something like that. Some from UX can confirm. @johanstromqvist do you remember us going over this from the new design language?

Reply:
I did it because it would create a gap between the items that @sarahill recommend removing.

<img width="934" alt="Screenshot 2023-06-01 at 1 40 10 PM" src="https://github.com/Shopify/polaris/assets/2091116/2ad7e8ae-50c7-40f8-8996-5722274f7c71">

Also, not sure how it would avoid the flickering because it was flickering with it and not without when I tested it 🤔

Reply:
Oh yeah we have flickering in production for some reason, must have been a regression at some point. If this matches figma then I'm fine, we can follow up if needed

Reply:
Oh yeah we have flickering in production for some reason, must have been a regression at some point. If this matches figma then I'm fine, we can follow up if needed